### ISSUE TITLE
Suggestion: replace (??) with <++> in vim for MerlinDestruct

### ISSUE THREAD

**Issue by msprotz** [ocaml/merlin#755]
This would allow jumps (via ctrl-j), a neat feature that jumps the cursor in edit mode directly to where the next <++> was.

**Comment by Leonidas-from-XIV** [comment]
Is that a plugin? Because in my vim `<C-j>` inserts a new line and that is the default unless I'm reading [this post](https://stackoverflow.com/q/29524398) wrong.

**Comment by msprotz** [comment]
Indeed, it looks like it's a customization from vim-latexsuite that's actually enabled by default, not just when editing tex files.

**Comment by msprotz** [comment]
I don't think it's worth adding complexity for this. I like it, but since it's not a standard feature (I thought it was) it's probably a fairly niche use-case. I'm fine with you closing this Thomas. Thanks!

**Comment by rgrinberg** [comment]
What I think would be better in fact is to turn destruct into an LSP snippet expansion. This would let you jump from clauses as you expect and it will work for editor backends supporting snippets.

**Comment by voodoos** [comment]
This issue is obsolete: now Merlin uses `_` instead of `(??)`.
